From 53bf6a597864ffa6789c1299af6e85edcb51979b Mon Sep 17 00:00:00 2001
From: Alex Leibovich <alexl@marvell.com>
Date: Mon, 18 May 2020 18:03:29 +0300
Subject: [PATCH 1151/1239] configs/mvebu: remove mvebu_db_ep_cn91xx_defconfig

Removed mvebu_db_ep_cn91xx_defconfig file.
Platforms running PCIe in EP mode will use
the mvebu_db_cn91xx_defconfig instead,
which solves the issue of simultaneous usage of
EP and RC PCIe modes in same setup

Change-Id: Id95314f9a82423da425734c5190a53095e37df80
Signed-off-by: Alex Leibovich <alexl@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/boot/u-boot/+/28632
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Kostya Porotchkin <kostap@marvell.com>
---
 configs/mvebu_db_ep_cn91xx_defconfig | 101 ---------------------------
 1 file changed, 101 deletions(-)
 delete mode 100644 configs/mvebu_db_ep_cn91xx_defconfig

diff --git a/configs/mvebu_db_ep_cn91xx_defconfig b/configs/mvebu_db_ep_cn91xx_defconfig
deleted file mode 100644
index 9a6cd1fd40..0000000000
--- a/configs/mvebu_db_ep_cn91xx_defconfig
+++ /dev/null
@@ -1,101 +0,0 @@
-CONFIG_ARM=y
-CONFIG_ARCH_MVEBU=y
-CONFIG_SYS_TEXT_BASE=0x00000000
-CONFIG_SYS_MALLOC_F_LEN=0x2000
-CONFIG_TARGET_OCTEONTX2_CN913x=y
-CONFIG_DEBUG_UART_BASE=0xf0512000
-CONFIG_DEBUG_UART_CLOCK=200000000
-CONFIG_MVEBU_SAR=y
-CONFIG_SMBIOS_PRODUCT_NAME=""
-CONFIG_DEBUG_UART=y
-CONFIG_AHCI=y
-CONFIG_DISTRO_DEFAULTS=y
-# CONFIG_SYS_MALLOC_CLEAR_ON_INIT is not set
-CONFIG_BOOTCOMMAND="run get_images; run set_bootargs; booti $kernel_addr $ramfs_addr $fdt_addr"
-CONFIG_SYS_CONSOLE_ENV_OVERWRITE=y
-CONFIG_SYS_CONSOLE_INFO_QUIET=y
-CONFIG_PL011_SERIAL=y
-CONFIG_CONSOLE_MUX=y
-CONFIG_OCTEONTX_SERIAL_PCIE_CONSOLE=y
-CONFIG_OCTEONTX_SERIAL_BOOTCMD=y
-# CONFIG_DISPLAY_CPUINFO is not set
-CONFIG_ARCH_EARLY_INIT_R=y
-CONFIG_ARCH_MISC_INIT=y
-CONFIG_BOARD_EARLY_INIT_F=y
-CONFIG_SYS_PROMPT="Marvell>> "
-CONFIG_CMD_MVEBU_SAR=y
-# CONFIG_CMD_FLASH is not set
-CONFIG_CMD_FUSE=y
-CONFIG_CMD_GPIO=y
-CONFIG_CMD_I2C=y
-CONFIG_CMD_MMC=y
-CONFIG_CMD_MTD=y
-CONFIG_CMD_PCI=y
-CONFIG_CMD_SF=y
-CONFIG_CMD_SPI=y
-CONFIG_CMD_USB=y
-# CONFIG_CMD_SETEXPR is not set
-CONFIG_CMD_CACHE=y
-CONFIG_CMD_TIME=y
-CONFIG_CMD_MVEBU_BUBT=y
-CONFIG_CMD_MVEBU_HW_INFO=y
-CONFIG_CMD_MVEBU_RX_TRAINING=y
-CONFIG_CMD_REGULATOR=y
-CONFIG_CMD_EXT4_WRITE=y
-CONFIG_CMD_MTDPARTS=y
-CONFIG_CMD_UBI=y
-CONFIG_MAC_PARTITION=y
-CONFIG_ENV_IS_IN_SPI_FLASH=y
-CONFIG_AHCI_MVEBU=y
-CONFIG_MVEBU_EFUSE=y
-CONFIG_MVEBU_EFUSE_READ_ONLY=y
-CONFIG_DM_GPIO=y
-CONFIG_DM_PCA953X=y
-CONFIG_DM_I2C=y
-CONFIG_SYS_I2C_MVTWSI=y
-CONFIG_MISC=y
-CONFIG_DM_MMC=y
-CONFIG_MMC_IO_VOLTAGE=y
-CONFIG_MMC_UHS_SUPPORT=y
-CONFIG_MMC_HS200_SUPPORT=y
-CONFIG_MMC_SDHCI=y
-CONFIG_MMC_SDHCI_XENON=y
-CONFIG_MTD=y
-CONFIG_NAND=y
-CONFIG_NAND_PXA3XX=y
-CONFIG_SPI_FLASH=y
-CONFIG_SPI_FLASH_MACRONIX=y
-CONFIG_SPI_FLASH_SPANSION=y
-CONFIG_SPI_FLASH_STMICRO=y
-CONFIG_SPI_FLASH_WINBOND=y
-CONFIG_SPI_FLASH_MTD=y
-CONFIG_PHY_MARVELL=y
-CONFIG_PHY_GIGE=y
-CONFIG_E1000=y
-CONFIG_MVPP2=y
-CONFIG_PCI=y
-CONFIG_DM_PCI=y
-CONFIG_MVEBU_COMPHY_SUPPORT=y
-CONFIG_PINCTRL=y
-CONFIG_PINCTRL_ARMADA_8K=y
-CONFIG_DM_REGULATOR_FIXED=y
-CONFIG_DM_REGULATOR_GPIO=y
-CONFIG_DM_RTC=y
-CONFIG_MARVELL_RTC=y
-CONFIG_DEBUG_UART_SHIFT=2
-CONFIG_DEBUG_UART_ANNOUNCE=y
-CONFIG_SYS_NS16550=y
-CONFIG_KIRKWOOD_SPI=y
-CONFIG_DM_THERMAL=y
-CONFIG_MVEBU_THERMAL_SENSOR_28NM=y
-CONFIG_MVEBU_THERMAL_EXT_SENSOR_28NM=y
-CONFIG_USB=y
-CONFIG_DM_USB=y
-CONFIG_USB_XHCI_HCD=y
-CONFIG_USB_EHCI_HCD=y
-CONFIG_USB_HOST_ETHER=y
-CONFIG_USB_ETHER_ASIX=y
-CONFIG_USB_ETHER_MCS7830=y
-CONFIG_USB_ETHER_RTL8152=y
-CONFIG_USB_ETHER_SMSC95XX=y
-CONFIG_SMBIOS_MANUFACTURER=""
-- 
2.29.0

